eclair-core/src/main/scala/fr/acinq/eclair/wire/internal/channel/version0/ChannelCodecs0.scala

```diff
@@ -69,7 +68,7 @@ private[channel] object ChannelCodecs0 {
         ("channelPath" | keyPathCodec) ::
         ("dustLimit" | satoshi) ::
         ("maxHtlcValueInFlightMsat" | uint64) ::
-        ("channelReserve" | satoshi) ::
+        ("channelReserve" | conditional(included = true, satoshi)) ::
```

Comment:
I suppose you have used the `conditional` codec here just for consistency?

Reply:
Yes, but I also don't see how else you would do it?

Reply:
Oh that's an optional field, right.